### PR TITLE
Intdeliver 311 downloads latest version of and platform specific bridge kit

### PR DESCRIPTION
**Pull request by kishorikumar**

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -18,23 +19,40 @@ export async function run() {
...
-          await rmRF(file)
-        }
+    let bridgeUrl = ''
+
+    const sb = new SynopsysBridge()
+    if (inputs.BRIDGE_DOWNLOAD_VERSION) {
+      if (await sb.validateBridgeVersion(inputs.BRIDGE_DOWNLOAD_VERSION)) {
+        bridgeUrl = sb.getVersionUrl(inputs.BRIDGE_DOWNLOAD_VERSION.trim()).trim()
+      } else {
+        return Promise.reject(new Error('The provided bridge version '.concat(inputs.BRIDGE_DOWNLOAD_VERSION).concat(' is not available in artifactory')))
```
Comment (tithiSynopsys) [inline src/main.ts:29]: Message can be-" Provided bridge version <version> not found in antifactory"

File: src/synopsys-action/synopsys-bridge.ts [hunk 2]
```diff
@@ -61,8 +67,75 @@ export class SynopsysBridge {
 
     return -1
   }
+
+  async getLatestVersion(): Promise<string> {
+    const versionArray: string[] = await this.getAllAvailableBridgeVersions()
```
Comment (tithiSynopsys) [inline src/synopsys-action/synopsys-bridge.ts:72]: once latest tag is attached to bridge package from bridge side this code not needed right?

File: src/main.ts (hunk 1 above)
Comment (kishorikumar) [reply src/main.ts]: Done.

File: src/synopsys-action/synopsys-bridge.ts (hunk 2 above)
Comment (kishorikumar) [reply src/synopsys-action/synopsys-bridge.ts:72]: Yes

**Review by tithiSynopsys — APPROVED** [review]

**Pull request merged**